tests/functional/docs/test_generate.py

```diff
@@ -33,3 +66,23 @@ def test_select_limits_no_match(self, project):
         run_dbt(["run"])
         catalog = run_dbt(["docs", "generate", "--select", "my_missing_model"])
         assert len(catalog.nodes) == 0
+
+    def test_catalog_with_sources(self, project):
+        run_dbt(["build"])
+        catalog = run_dbt(["docs", "generate"])
+
+        # 2 seeds + 2 models
+        assert len(catalog.nodes) == 4
+        # 2 sources (only ones that exist)
+        assert len(catalog.sources) == 2
+
+    def test_select_source(self, project):
+        run_dbt(["build"])
+        catalog = run_dbt(["docs", "generate", "--select", "source:test.my_seed.sample_seed"])
+
+        # 2 seeds
+        # TODO: Filtering doesn't work for seeds
```

Comment:
It seems odd to assert behavior we regard as incorrect. Do you think it's valuable enough to leave in? If so, can you add some notes about what the correct values would be?

Reply:
@aranke do we have a ticket for these `TODO`s yet? If so, could you share the link here?